### DISCUSSION TITLE
starlink是解決交通雜亂的關鍵？

### DISCUSSION THREAD

**Discussion by alvin1226** [1101-GS3073-C/Python-Community-GS3073-C#686] [General]
大家有聽過starlink嗎？是SpaceX發射出的幾萬顆衛星計畫
現在已經可以透過starlink來定位tesla，而在未來有可能藉由定位結合龐大的運算資料，我們是不是就有可能實現之前提到的理想城市中，所有車子都可以在AI的運算內，行駛在計算好的路徑上，這樣便不會有車禍的出現了！
https://www.inside.com.tw/article/29503-spacex-starlink-internet

**Comment by 0212Jerry** [comment]
我覺得有機會，但等到其蓬勃發展還需要幾年的時間，衛星連網需要穩定。而且在硬體上，每一輛汽車或摩托車等都需要可以支援衛星網路，光這點要普及就需要五年以上。而車禍我認為還是有可能發生，因為行人與腳踏車等慢車行蹤難以掌控，對於突發狀況還是沒辦法到完全掌握。

**Comment by Vincent20030314** [comment]
確實是不錯的發明，但是應該還需要很多年的時間發展。

**Comment by jerry0108566** [comment]
之前在一個國外頻道CGP Grey的一部影片上有看到，說造成塞車的主要原因其實在於協調性，因為每個駕駛人的反應速度都不相同，限制了能通過路口的車流量，因此造成交通不協調。若是這項技術能夠完善，那在未來也就不會遇到塞車了。
影片：[https://www.youtube.com/watch?v=iHzzSao6ypE](https://www.youtube.com/watch?v=iHzzSao6ypE)

**Comment by HsuChen88** [comment]
從另一個角度來說，AI需要儲存的數據非常龐大，應該也要在資料儲存的技術上突破，才能解決不斷興建資料庫的窘境。

**Comment by yjpxi** [comment]
儘管Starlink 提供了高速連接，但在實際應用中仍可能面臨數據延遲和處理能力的挑戰，自動駕駛系統需要即時反應，任何延遲都可能影響安全。
